setup/install/environment.md

```diff
@@ -67,6 +67,54 @@ hal config deploy edit --type distributed --account-name $ACCOUNT
 This command changes the type of the next deployment of Spinnaker, and will
 deploy it to the account you have previously configured.
 
+## Local Git
+
+The __Local Git__ installation means Spinnaker will be cloned, built and run on
+the single machine Halyard is run on.
+
+### Intended Use-case
+
+The __Local Git__ installation is intended for developers who want to contribute
+to Spinnaker. It is not intended to be used to manage any production environment.
+
+### Prerequisites
+
+#### Install local dependancies
+
+Ensure that the following are installed on your system
+
+* git
+* curl
+* yarn ([guide](https://yarnpkg.com/lang/en/docs/install/))
+
+#### Fork all Spinnaker repos
+
+Fork all of the microservices listed here: [Spinnaker Microservices](https://www.spinnaker.io/reference/architecture/#spinnaker-microservices) on github ([guide](https://guides.github.com/activities/forking/#fork)).
+
+#### Setup SSH Keys
+
+Follow the following guides to setup ssh access to your github.com account from your local machine.
```

Comment:
Colon at end of this sentence. 
Also, either "Use the following guides..." or "Follow these guides..."

Reply:
Done.